sdk/core/azure-core/inc/azure/core/context.hpp

```diff
@@ -197,15 +199,15 @@ namespace Azure { namespace Core {
      */
     void Cancel()
     {
-      m_contextSharedState->CancelAtMsecSinceEpoch
-          = ContextSharedState::ToMsecSinceEpoch((time_point::min)());
+      m_contextSharedState->Expiration
+          = ContextSharedState::ToDateTimeRepresentation((DateTime::min)());
     }
 
     /**
      * @brief Check if the context is cancelled.
      * @return `true` if this context is cancelled, `false` otherwise.
      */
-    bool IsCancelled() const { return CancelWhen() < std::chrono::system_clock::now(); }
+    bool IsCancelled() const { return GetExpiration() < std::chrono::system_clock::now(); }
```

Comment:
Can we compare against DateTime::now and avoid chrono

Reply:
Our `Azure::DateTime` does not have `Now()`. `std::chrono::system_clock` has, it returns `std::chrono::system_clock::time_point`, which can be implcitly converted to `Azure::DateTime`.